include/pico_stack.h

```diff
@@ -16,14 +16,42 @@
 
 /* ===== RECEIVING FUNCTIONS (from dev up to socket) ===== */
```

Comment:
shouldn't this line also be removed? Looks like the functions are now sorted by layer first...